docs/solarsystem/pds/pds.rst

```diff
@@ -0,0 +1,116 @@
+.. _astroquery.solarsystem.pds:
+
+***********************************************************************************
+PDS Planetary Ring Node Queries (`astroquery.solarsystem.pds`)
+***********************************************************************************
+
+Overview
+========
+
+
+The :class:`~astroquery.solarsystem.pds.RingNodeClass` provides an
+interface to the ephemeris tools provided by the `NASA Planetary Data System's Ring Node System`_ hosted by SETI institute.
+
+Ephemeris
+-----------
+
+In order to query information for a specific Solar System body, a
+``RingNode`` object is instantiated and the :meth:`~astroquery.solarsystem.pds.RingNodeClass.ephemeris` method is called. The following example queries the
+ephemerides of the rings and small moons around Uranus as viewed from ALMA:
+
+.. code-block:: python
```

Comment:
you need to do all these snippets in a `doctest-remote-data` context, so they are skipped for local only tests. The context need to be added for each and every snippet (we don't yet support adding only one for the top of the file).